qiita_db/test/test_metadata_template.py

```diff
@@ -652,6 +693,48 @@ def test_create(self):
                ['2.Sample3', "Value for sample 3"]]
         self.assertEqual(obs, exp)
 
+    def test_create_already_prefixed_samples(self):
```

Comment:
Can you add a test case for samples that are prefixed with a number that doesn't match the study_id and one where the prefixes are different numbers/strings?